src/cbmc/CMakeLists.txt

```diff
@@ -58,15 +58,15 @@ endif()
 
 # bash completion
 if(NOT WIN32)
-  add_custom_command(OUTPUT "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/cbmc.sh"
-    COMMAND "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/extract_switches.sh"
+  add_custom_command(OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/../../scripts/bash-autocomplete/cbmc.sh"
+    COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/../../scripts/bash-autocomplete/extract_switches.sh"
```

Comment:
I want to defer checking the correctness of that to @esteffin

Reply:
This solves the problem, however I would suggest a tidier approach that avoids the `../..`.

Why not adding a variable `CBMC_ROOT_DIR` to the `CMakeLists.txt` with the value of `${CMAKE_CURRENT_SOURCE_DIR}`, and then refer to that from `src/cbmc/CMakeLists.txt`?

This will remove all the `..`s and also will be usable from other folders in case it is needed.

Reply:
Thanks for the suggestion, I changed it to your suggestion.